game/src/main/java/game/action/CreateBulletAction.java

```diff
@@ -0,0 +1,31 @@
+package game.action;
+
+import engine.action.Action;
+import engine.geometry.Rectangle;
+import engine.geometry.Vector;
+import engine.object.manager.ObjectManager;
+import game.object.Bullet;
+import game.object.GameObjectFactory;
+
+public class CreateBulletAction implements Action {
+
+    private final ObjectManager<Bullet> bulletObjectManager;
+    private final GameObjectFactory gameObjectFactory;
+    private final Rectangle bulletBody;
+    private final Vector speed;
+
+    public CreateBulletAction(ObjectManager<Bullet> bulletObjectManager, GameObjectFactory gameObjectFactory,
```

Comment:
Hm... It seems that its better to name it ShootBulletAction?  Because you dont just create but also accelerate it